test/Streamly/Test/Prelude/WSerial.hs

```diff
@@ -64,6 +79,29 @@ main = hspec
     describe "Monoid operations" $ do
         wSerialOps   $ monoidOps "wSerially" mempty sortEq
 
+    describe "Bind and Monoidal composition combinations" $ do
+        -- XXX Taking a long time when wSerialOps is used.
+        bindAndComposeSimpleOps "WSerial" sortEq wSerially
+        bindAndComposeHierarchyOps "WSerial" wSerially
+        wSerialOps $ nestTwoStreams "WSerial" id sort
+        wSerialOps $ nestTwoStreamsApp "WSerial" id sort
+        composeAndComposeSimpleSerially
+            "WSerial <> "
+            [ [1, 4, 2, 7, 3, 5, 8, 6, 9]
+            , [1, 7, 4, 8, 2, 9, 5, 3, 6]
+            , [1, 4, 2, 7, 3, 5, 8, 6, 9]
+            , [1, 7, 4, 8, 2, 9, 5, 3, 6]
+            ]
+            wSerially
+        composeAndComposeSimpleWSerially
+            "WSerial <> "
+            [ [1, 4, 2, 7, 3, 5, 8, 6, 9]
+            , [1, 7, 4, 8, 2, 9, 5, 3, 6]
+            , [1, 4, 3, 7, 2, 6, 9, 5, 8]
+            , [1, 7, 4, 9, 3, 8, 6, 2, 5]
+            ]
+            wSerially
```

Comment:
Did you miss `composeAndComposeSimpleWSerially` `serially`

Reply:
It's in Prelude.Serial